ensemble/ctf/window_function_component.py

```diff
@@ -98,7 +98,7 @@ def values(self):
 
         xs = np.linspace(center - radius, center + radius, num_samples)
         ys = self.window_func(num_samples) * self.opacity
-        return zip(xs, ys)
+        return list(zip(xs, ys))
```

Comment:
Is it actually a problem if this returns an iterator?

Reply:
If we return an iterator here, we will need to wrap the call elsewher

See https://github.com/enthought/ensemble/blob/66b23c72227d880c28cb17615d3044242ac33511/ensemble/ctf/editor.py#L176

Reply:
Not really though. The code you linked to is using the return value of `values` in a way which is compatible with an iterator. (unless I'm missing something)

Reply:
Ohh. yes. sorry. `alpha_nodes` is iterated over so we don't have to do `list(zip(...))`.
I guess my main problem is with the difference in behavior of `values` between the various `*Node` objects.

Reverting this change for now.